src/styles/reset.scss

```
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+p {
+  margin: 0;
+}
```

There must be an empty line at the end of the file

```suggestion
}

```